tests/cross/src/test/scala/tests/pc/TypeDefinitionSuite.scala

```diff
@@ -0,0 +1,39 @@
+package tests.pc
+
+object TypeDefinitionSuite extends BaseTypeDefinitionSuite {
+
+  check("val")(
+    """
+      |class Main(i: Int) {
+      |
+      |}
+      |
+      |object Main {
+      |  val ts@@t = new Main(2)
+      |}""".stripMargin,
+    "Main.scala [2:6 -> 2:9]"
+  )
+
+  check("constructor")(
+    """
+      |class Main(i: Int) {}
+      |
+      |object Main {
+      | def tst(m: Main): Unit = {}
+      |
+      |  tst(new M@@ain(2))
+      |}""".stripMargin,
+    "Main.scala [2:6 -> 2:9]"
+  )
+
+  check("method")(
+    """
+      |object Main {
+      |  def tst(): Unit = {}
+      |
+      |  ts@@t()
+      |}""".stripMargin,
+    "Main.scala [3:6 -> 3:8]"
```

Comment:
I would expect this to go to the definition of `Unit` instead of `tst()`

Reply:
Changed.